### ISSUE TITLE
OrderByPipe should check if Array exists

### ISSUE THREAD

**Issue by gtsopour** [FuelInteractive/fuel-ui#93]
```
transform(input: any, config: string = '+'): any {
        if (input) {
        }
}

```

It seems that into the transform you need to check if array exists. Could you please verify that?

Best
George

**Comment by coryshaw1** [comment]
A check similar to this will be added to the next update. Will keep this issue open until the update is published.

**Comment by coryshaw1** [comment]
Closed with #90 and npm package 0.3.8
